docs/source/tensorbay_cli/getting_started_with_cli.rst

```diff
@@ -105,3 +105,28 @@ CLI: Commit the Draft
 .. code:: html
 
    $ gas commit tb:<dataset_name>#<draft_number> [-m <message>]
+
+***************
+Profile
+***************
+
+Set accessKeys into different profiles to distinguish different application scenarios,
```

Comment:
`For users with multi TensorBay accounts or different workspaces, CLI provides profiles to easily authenticate and use different accesskeys.`?